api/users/models.py

```diff
@@ -156,6 +156,11 @@ def join_organisation_from_invite_link(self, invite_link: "InviteLink"):
 
     def join_organisation_from_invite(self, invite: "AbstractBaseInviteModel"):
         organisation = invite.organisation
+        susbcription_metadata = organisation.subscription.get_subscription_metadata()
+
+        if invite.organisation.num_seats >= susbcription_metadata.seats:
+            organisation.subscription.add_single_seat()
```

Comment:
This feels a _little_ dark to me. Especially since invite links don't have an expiry, if a malicious actor got hold of it they could subscribe as many times as they liked and cost an organisation a serious amount of money. At least with invite emails they are tied to an email and can be revoked, etc. Should we prevent invite links from automatically adding seats? 🤔 

As part of this work, in general, does it not make sense to have an endpoint to add a seat to an organisation? Then we can show an error when someone tries to sign up via an invite link, directing them to talk to their org admin to add more seats. 

Thoughts @dabeeeenster ?

Reply:
This never expiring invite link also feels a bit wrong though

Reply:
Yeah, well they technically can have an expiry but the FE doesn't currently expose it. I still think that exposing a button in the UI for org administrators is the way forward here. Let's discuss with @dabeeeenster .

Reply:
By exposing a button, do you mean to invalidate an invite link ? We did add a button to regenerate/invalidate an invite link if so.
![image](https://user-images.githubusercontent.com/8608314/207598811-168a1aa8-73e9-48c6-b18f-5254ced14429.png)

Reply:
That helps, but invite links can also have an optional expiry datetime - which the current UI pattern doesn't really support.

Reply:
Resolving this based on conversation in Slack. Consensus is to stick with the current approach and automatically add seats.